src/lib/types/events.ts

```diff
@@ -82,6 +82,8 @@ export const PUBLIC_SIGNUP_TOKEN_CREATED = 'public-signup-token-created';
 export const PUBLIC_SIGNUP_TOKEN_USER_ADDED = 'public-signup-token-user-added';
 export const PUBLIC_SIGNUP_TOKEN_TOKEN_UPDATED = 'public-signup-token-updated';
 
+export const SUGGEST_CHANGE_CREATED = 'suggest-change-created';
```

Comment:
Nice! Maybe I can ask you to add this to the docs too? A short description of when it fires and what it is in `website/docs/api/admin/events-api.md`? Along with a note that it is an experimental feature? (I'll make it so that we can feature gate it in the future!)